NotificationsPipeline.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+import pytz, importlib, logging
+import NotificationsMessage as nm
+# Imports for all the pipeline steps
+from notifications_pipeline_steps.NotificationsPipelineUserTargeting import NotificationsPipelineUserTargeting
+from notifications_pipeline_steps.NotificationsPipelineUserPolicy import NotificationsPipelineUserPolicy
+from notifications_pipeline_steps.NotificationsPipelinePresentation import NotificationsPipelinePresentation
+from notifications_pipeline_steps.NotificationsPipelineDelivery import NotificationsPipelineDelivery
+
+def import_from_string(module_name):
+    # Remove during refactoring
+    class_name = module_name.split(".")[-1]
+    return getattr(importlib.import_module(module_name), class_name)
+
+class NotificationsPipeline(object):
+    def __init__(self):
+        self.pipeline_step_list = [
+        "NotificationsPipelineComposer",
+        "notifications_pipeline_steps.NotificationsPipelineUserTargeting",
+        "notifications_pipeline_steps.NotificationsPipelineUserPolicy",
+        "notifications_pipeline_steps.NotificationsPipelinePresentation",
+        "notifications_pipeline_steps.NotificationsPipelineDelivery"
+        ]
+    def on_enter(self, message):
```

Comment:
I called it pipeline_process because process_message is the name of the method that each pipeline step implements for processing the message at that step.